glib-macros/src/lib.rs

```diff
@@ -445,7 +446,7 @@ pub fn closure_local(item: TokenStream) -> TokenStream {
 /// use glib::subclass::prelude::*;
 ///
 /// #[derive(Debug, Copy, Clone, PartialEq, Eq, glib::Enum)]
-/// #[enum_type(name = "MyEnum")]
+/// #[enum_type(name = "MyEnum", nullable)]
```

Comment:
I'd expect this to fail. It should :)

Reply:
Except it doesn't fail. The doc check failed for other network related reasons.

Reply:
Yeah it doesn't check if there are random other, unknown attributes apparently.

Reply:
https://github.com/gtk-rs/gtk-rs-core/issues/943

Reply:
So just remove the wrong attribute here for now :)